test/parsing/stringLiteralIndex/indexWithNonParam.good

```diff
@@ -0,0 +1,10 @@
+h
+hape
+hapel
+Cha
+ha
+104
```

Comment:
I'm ashamed to admit it, because I know I advocated strongly for it, but I had to think for far too long about why this is `104` rather than "h"...

Reply:
FWIW, I have no regrets :) It is probably more confusing in this context than it actually is in a more realistic one.